mix.exs

```diff
@@ -53,7 +53,7 @@ defmodule ExGram.Mixfile do
   defp deps do
     [
       # Tesla adapter
-      {:tesla, "~> 1.2", optional: true},
```

Comment:
I think that Tesla 1.2 works perfectly with the changes, I would like to keep the minimum version to `1.2` and not update any dependency version either

Reply:
[keep the same tesla version](https://github.com/rockneurotiko/ex_gram/pull/185/commits/29e404bab39b2b5a5c81599f938a7457382bae79)